### ISSUE TITLE
Weird markdown indentation issue

### ISSUE THREAD

**Issue by frioux** [sheerun/vim-polyglot#270]
This is so weird.  This happens for `vim-polyglot` but *not* the underlying `vim-markdown`.

Make a vimrc like this:

``` vim
set rtp+=~/code/vim-polyglot
" Note that this filetype toggle happens when you use pathogen
filetype off
filetype on
filetype plugin on
filetype plugin indent on
syntax enable
```

Edit this `test.md` file:

``` markdown
---

 * foo
   * bar

```

Then type: `/bar<CR>A<CR>`

Adding the newline, bizarrely, **de-indents** the ` * bar`.  What.

**Comment by idbrii** [comment]
polyglot only contains a [syntax file for markdown](https://github.com/sheerun/vim-polyglot/blob/master/syntax/markdown.vim). Since there's no ftplugin, this behavior is not because polyglot breaks something, it's because it doesn't add it! vim-markdown provides a [markdown indent file](https://github.com/plasticboy/vim-markdown/blob/a3169545f330ec8080244c3ad755bb2211eca8a0/indent/markdown.vim) so it does special indent handling for markdown. polyglot does not.

The vimrc you're providing is not setting up any indent behavior, so vim does the dumb thing and provides none. I'd recommend doing barebones tests like this with [sensible](https://github.com/tpope/vim-sensible) so you have something close to a realistic workspace. That would give you `set autoindent` which fixes the issue.

polyglot claims:

> Solid syntax and indentation support (other features skipped). Only the best language packs.

So maybe it's a bug that indent support is not included?

**Comment by frioux** [comment]
good call re `~/.vim`, will check that when I get a chance.

**Comment by sheerun** [comment]
vim-polyglot includes syntax, indent and compiler configuration

this is indeed a bug of not including indentation for markdown that I fixed in v3.3.2

Thank you for reporting!

**Comment by frioux** [comment]
This is fixed; what a relief!
